samples/star-wars-api/Startup.fs

```diff
@@ -29,9 +29,10 @@ type Startup private () =
             log.LogError(EventId(), ex, "An unhandled exception has occurred while executing the request.")
             clearResponse >=> setStatusCode 500
 
-        if env.IsDevelopment() then
-            app.UseGraphQLPlayground("/playground") |> ignore
-            app.UseGraphQLVoyager("/voyager") |> ignore
+        app.UseGraphQLPlayground("/playground") |> ignore
+        app.UseGraphQLVoyager("/voyager") |> ignore
+        app.UseRouting() |> ignore
+        app.UseEndpoints(fun endpoints -> endpoints.MapBananaCakePop(new Microsoft.AspNetCore.Http.PathString("/cakePop")) |> ignore) |> ignore
```

Comment:
Move `Microsoft.AspNetCore.Http` to opens
Let's demonstrate here how to enable it for development only. If someone decides to publish that to production he/she will change that himself
```suggestion
        if env.IsDevelopment() then
            app.UseGraphQLPlayground("/playground") |> ignore
            app.UseGraphQLVoyager("/voyager") |> ignore
        app.UseRouting() |> ignore
        app.UseEndpoints(
            fun endpoints ->
                if env.IsDevelopment() then endpoints.MapBananaCakePop(PathString("/cakePop")) |> ignore
                else ()) |> ignore
```